src/content/developers/tutorials/run-light-node-geth/index.md

```diff
@@ -1,173 +1,82 @@
 ---
 title: How to run a light node with Geth
-description: How to download, install and run Geth. Covering syncmodes, the Javascript console, and more
-author: "Brian Gu"
+description: How to download, install and run Geth "light".
+authors: ["Brian Gu", "Chris Hobcroft"]
```

Comment:
Unfortunately, this is not possible with how our tutorials are set up at the moment.

```suggestion
authors: "Brian Gu"
```